flaky test testClearWithBiggerMinimumCapacityDoesNotDupeStorageIfTheresOnlyOneBuffer

this one just failed in macOS CI!?

```
13:03:38 - Test Case '-[NIOTests.ByteBufferTest testClearWithBiggerMinimumCapacityDoesNotDupeStorageIfTheresOnlyOneBuffer]' started.
13:03:38 - /otmp/od.WeNfjl/lw.rcuCL8/q6qm-nts078ati27iq_0-1412432/t.AEPNkU/0~q5iwr53_zrV9ndjAV8iBQWOtBJ2bDgtCXdNDh4ASvFKCMIV2YyO70neBDkFISMfmqKQQJDiT4UzQd_VMvEVaBg==/SRCROOT/Tests/NIOTests/ByteBufferTest.swift:1372: error: -[NIOTests.ByteBufferTest testClearWithBiggerMinimumCapacityDoesNotDupeStorageIfTheresOnlyOneBuffer] : XCTAssertNotEqual failed: ("140308679511184") is equal to ("140308679511184") - 
```

--- a/Tests/NIOTests/ByteBufferTest.swift
+++ b/Tests/NIOTests/ByteBufferTest.swift
@@ -1358,18 +1358,12 @@ class ByteBufferTest: XCTestCase {
         let alloc = ByteBufferAllocator()
         var buf = alloc.buffer(capacity: 16)
 
-        var bufPtrValPre: UInt = 1
-        var bufPtrValPost: UInt = 2
-
         XCTAssertLessThan(buf.capacity, 32)
         let preCapacity = buf.capacity
 
-        bufPtrValPre = buf.storagePointerIntegerValue()
         buf.clear(minimumCapacity: 32)
-        bufPtrValPost = buf.storagePointerIntegerValue()
         let postCapacity = buf.capacity
 
-        XCTAssertNotEqual(bufPtrValPre, bufPtrValPost)
         XCTAssertGreaterThanOrEqual(buf.capacity, 32)
         XCTAssertNotEqual(preCapacity, postCapacity)
     }

